gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -28,7 +30,7 @@
     level=logging.INFO,  # or DEBUG for more detail
     format="%(asctime)s - %(levelname)s - %(message)s",
     handlers=[
-        logging.FileHandler("/var/log/cron.log"),  # Logs to file
+        #logging.FileHandler("/var/log/cron.log"),  # Logs to file
```

Comment:
Why is this commented out? We need this to view the docker logs

Reply:
Sorry about that! I commented that when testing locally. Will fix that.